android/src/fdroid/play/listings/de-DE/title.txt

```diff
@@ -1 +1 @@
-Organic Maps Offline Hike, Bike & GPS Navigation
+Organic Maps Offline Wandern, Radfahren & GPS-Navigation
```

Comment:
"Organic Maps Offline Wandern, Radeln & Navigation" to fit the 50 character limit

Reply:
Updated with https://github.com/organicmaps/organicmaps/pull/3351/commits/8a956cf2a917acde71f2a3519f0e5ed77bccc0c6